src/components/hosts/HostsTable.tsx

```diff
@@ -215,8 +215,10 @@ const HostsTable: React.FC<HostsTableProps> = ({ cluster, skipDisabled = false }
     async (event: React.MouseEvent, rowIndex: number, rowData: IRowData) => {
       const hostId = rowData.host.id;
       try {
-        const { data } = await disableClusterHost(cluster.id, hostId);
-        dispatch(updateHost(data));
```

Comment:
But it will lead to potentially inconsistent state, i.e. a master host is `disabled` but the cluster is still `ready`. If we skip this update, the user will consider the delay of waiting for new cluster details as a part of the action.

Reply:
You're right. ok.